tools/scylla-sstable.cc

```diff
@@ -1451,6 +1452,22 @@ class scylla_metadata_visitor : public boost::static_visitor<> {
         _writer.String(disk_string_to_string(val));
     }
 
+    void operator()(const sstables::clustering_position_metadata& val) const {
+        auto write_pos = [this] (sstring_view name, const auto& v) {
+            _writer.Key(name);
+            _writer.StartArray();
+            for (const auto& e : v.pos.elements) {
+                _writer.String(to_sstring_view(e.value));
+            }
+            _writer.EndArray();
+        };
+        _writer.StartObject();
+        write_pos("first_clustering_position", val.first_clustering_position);
+        write_pos("last_clustering_position", val.last_clustering_position);
+        _writer.EndObject();
+
+    }
+
```

Comment:
Please also update the documentation of the json output: https://github.com/scylladb/scylladb/blob/996eac9569fba1b08ae15e2533a158fd0d23b214/tools/scylla-sstable.cc#L2862-L2895

Reply:
will do